src/translation-wrapper.js

```diff
@@ -32,14 +32,14 @@ function onsplit(payload, socketIndex) {
     if (!_.isObject(data)) throw Error('Translation wrapper requires a hash on the input port');
     var groupLm = payload.lm || createLm();
     _.each(_.pick(data, isAssignedToMe, this), function(element, vnid) {
-        ondata.call(this, createState(
-            vnid,
-            element.data,
-            createLm(), // Create an lm for the new vnis we will be sending
-            payload.error,
-            payload.stale,
-            groupLm
-        ), socketIndex);
+        var state =  createState( vnid,
+                                  element.data,
+                                  createLm(), // Create an lm for the new vnis we will be sending
```

Comment:
createLm() should be called only once, outside the _.each loop, so that the same LM is assigned to each new state.  See split-wrapper.js line 53: https://github.com/rdf-pipeline/noflo-rdf-components/blob/develop/src/split-wrapper.js#L53